Client/Frontend/Settings/HomePageSettingViewController.swift

```diff
@@ -41,6 +41,11 @@ class HomePageSettingViewController: SettingsTableViewController, FeatureFlagsPr
         return true
     }
 
+    var isHistoryHighlightsSectionEnabled: Bool {
+        // TODO: If this feature is going behind a Nimbus flag, that should be added here
```

Comment:
Ok so we don't know yet if it will be with nimbus or not?

Reply:
Negative. I assume it will be, but leaving it unknown until we... know (LOL) for sure.